src/main/java/com/spartronics4915/frc2019/subsystems/Drive.java

```diff
@@ -200,7 +202,20 @@ public void registerEnabledLoops(ILooper in)
     {
         in.register(mLoop);
     }
-
+    /**
+     * 
+     */
+    public synchronized void setWantTurn(Rotation2d heading)
+    {
+        if (mDriveControlState != DriveControlState.TURN)
+        {
+            logDebug("Switching to field relative turn: " + heading);
```

Comment:
This is actually a robot relative turn... If we want field relative turn we would have to turn to: `heading.inverse().rotateBy(mPeriodicIO.gyro_heading)`. We want a robot relative turn (because our gyro might drift) so just keep this as-is and change this `logDebug`.